service/src/main/java/com/epam/esm/service/exception/order/OrderAlreadyExistsException.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.esm.service.exception.order;
+
+import com.epam.esm.service.exception.ServiceException;
+
+public class OrderAlreadyExistsException extends ServiceException{
```

Comment:
You can use only one AlreadyExistsException (DuplicateEntityException) or NotFoundException with different error codes. For example:
NotFoundException("... 01") when Order not found
NotFoundException("... 02") when Tag not found
NotFoundException("... 03") when User not found